modules/govuk/files/node/s_jumpbox/amazon-cloudwatch-agent.json

```diff
@@ -0,0 +1,18 @@
+{
+  "agent": {
+    "run_as_user": "root"
```

Comment:
We may not need this, as the [default user should already be root](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html)